src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property should be moved to a global style file (like `main.scss` or `reset.scss`) using a universal selector (*), as it's a good practice to apply this property globally for consistency. It's not an error, just a recommendation for better code organization.